### DISCUSSION TITLE
How can i compile and upload code to atmega32, please provide more instructions..

### DISCUSSION THREAD

**Discussion by Pavljava** [Electrostat-Lab/AVR-Sandbox#69] [Q&A] [answered]
Could you please provide more instructions ?

**Comment by pavly-gerges** [comment, marked as answer]
Hi fellows, i have added #70 as a request for this, we can use those for now: 
- [AVR-Tools-Guide](https://software-hardware-codesign.github.io/AVR-Sandbox/docs/avr-libc/avr-libc-user-manual/using__tools.html)
- [HelloBlink Example](https://github.com/Software-Hardware-Codesign/AVR-Sandbox/tree/master/HelloBlink/build)
- [HelloBlink readme]()
